tests/core/test_proxy.py

```diff
@@ -44,3 +63,50 @@ def test_no_proxy(self):
         env.pop("https_proxy", None)
         env.pop("HTTP_PROXY", None)
         env.pop("HTTPS_PROXY", None)
+
+class TestProxy(TestCase):
+    @attr(requires='test')
```

Comment:
I don't think the test is running. I _think_ this should be `@attr(requires='core_integration')`. It doesn't look like `test_no_proxy` is running either actually.